[ EWS macOS WK1 ] run-layout-tests-in-stress-mode always runs tests in WK2 (with WKTR)

briannafan opened the pull request:
#### 6d5c08794a5719fc189eedd788b36df7fc9dc264
<pre>
[ EWS macOS WK1 ] run-layout-tests-in-stress-mode always runs tests in WK2 (with WKTR)
<a href="https://bugs.webkit.org/show_bug.cgi?id=261423">https://bugs.webkit.org/show_bug.cgi?id=261423</a>
rdar://115672166

Reviewed by Aakash Jain and Jonathan Bedard.

Sets flag for WK1 if LayoutTestClass == RunWebKit1Tests.

* Tools/CISupport/ews-build/factories.py:
(TestFactory.__init__):
* Tools/CISupport/ews-build/steps.py:
(RunWebKitTestsInStressMode.__init__):
(RunWebKitTestsInStressMode.setLayoutTestCommand):
* Tools/CISupport/ews-build/steps_unittest.py:
(TestRunWebKitTestsInStressMode.test_success_wk1):

Canonical link: <a href="https://commits.webkit.org/268254@main">https://commits.webkit.org/268254@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8015c6bdbc6ce656551de18c134afc61b2fa6651

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/18993 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/19338 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/19945 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/20862 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/17760 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/19181 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/22632 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/19538 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/19528 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/19218 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/19326 "1 flakes 4 failures") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/16537 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/21751 "Built successfully") | 
| | [❌ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/16535 "7 flakes 3 failures") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/17315 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/23713 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/17574 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/17487 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/21656 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/18061 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/15364 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/18826 "Passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/17144 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/21505 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/2332 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/17894 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aj062 reviewed (approved):
r+ assuming this is tested

briannafan committed:
[ EWS macOS WK1 ] run-layout-tests-in-stress-mode always runs tests in WK2 (with WKTR)
https://bugs.webkit.org/show_bug.cgi?id=261423
rdar://115672166

Reviewed by Aakash Jain and Jonathan Bedard.

Sets flag for WK1 if LayoutTestClass == RunWebKit1Tests.

* Tools/CISupport/ews-build/factories.py:
(TestFactory.__init__):
* Tools/CISupport/ews-build/steps.py:
(RunWebKitTestsInStressMode.__init__):
(RunWebKitTestsInStressMode.setLayoutTestCommand):
* Tools/CISupport/ews-build/steps_unittest.py:
(TestRunWebKitTestsInStressMode.test_success_wk1):

Canonical link: https://commits.webkit.org/268254@main
